docs/archives/backend.md

```diff
@@ -0,0 +1,44 @@
+
+When fetching article text on the frontend (e.g. when you go to an article and see the corresponding text to that article in the side bar), this text comes from the archives-text repository. The backend is really all handled by github and their API. The only thing that concerns us here is ensuring that this repo's data is up to date.
+
+## Responding to Frontend Fetching Issue Images
+Issue images are fetched on the frontend by openseadragon. These images live in AWS s3 buckets, so the images are fetched from there. I don't know much about this tbh so I can't really document it well
+
+## Responding to Frontend Submitting Corrections to Articles
+The frontend corrections form is powered by google forms. We're using an embedded form. Checkout [this](https://trevorfox.com/2015/06/dynamically-pre-fill-google-forms-with-mailchimp-merge-tags/) for more details
+
+## Cloudsearch
+Cloudsearch is a search engine service by AWS which powers the archives search functionality. Cloudsearch is powered by Apache Solr. 
+
+To "see" cloudsearch, go to `aws.amazon.com` and sign in. Then, in the find services search bar, search for cloudsearch. Once there, there should be a `archives-text-cloudsearch` domain. If you click on it, you can interact with the cloudsearch through aws's interface. Here you can search, and you can upload data. Note that you don't need to use AWS's own interface to do these two things. Amazon provides APIs which we can do these things programmatically. 
+
+We interact with cloudsearch programmatically in two ways:
+
+- Searching
+- Uploading Data
+
+### Searching
+Searching is pretty simple. You just submit a query to an endpoint and the endpoint responds with the search results. 
+
+### Uploading Data
+We have around 10gb of article text data in the archives-text repository and we need to upload that data to cloudsearch so that it has the data which we are searching over. There are python scripts which upload this data to cloudsearch in the archives-scripts repository. You can find them [here](https://github.com/thestanforddaily/archives-scripts) and [here](https://github.com/TheStanfordDaily/archives-scripts/tree/master/cloudsearch). These two sources have a bit more doc about cloudsearch, so go there to find out more.
```

Comment:
```suggestion
We have around 10GB of article text data in the archives-text repository and we need to upload that data to CloudSearch so that it has the data which we are searching over. There are python scripts which upload this data to CloudSearch in the archives-scripts repository. You can find them [here](https://github.com/thestanforddaily/archives-scripts) and [here](https://github.com/TheStanfordDaily/archives-scripts/tree/master/cloudsearch). These two sources have a bit more documentation about CloudSearch, so go there to find out more.
```